core/src/test/java/org/apache/iceberg/rest/TestHTTPClient.java

```diff
@@ -71,6 +71,8 @@ public class TestHTTPClient {
 
   private static final int PORT = 1080;
   private static final String BEARER_AUTH_TOKEN = "auth_token";
+  private static final String USER_AGENT = "User-Agent";
```

Comment:
nit: `USER_AGENT`  constant exists in `HttpHeaders` class.

Reply:
I was thinking about adding this constant to class `HTTPClient`, but if it's only used by test, we might not do that.